pyo3-build-config/src/lib.rs

```diff
@@ -153,6 +153,10 @@ pub fn print_feature_cfgs() {
     if rustc_minor_version >= 74 {
         println!("cargo:rustc-cfg=invalid_from_utf8_lint");
     }
+
+    if rustc_minor_version >= 75 {
+        println!("cargo:rustc-cfg=clippy_unnecessary_fallible_conversions_lint");
+    }
```

Comment:
I think this bit can be reverted, otherwise please merge 👍

Reply:
Done